specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/importExport.json

```diff
@@ -0,0 +1,552 @@
+          },
+          "202": {
+            "description": "Importing the database is in progress."
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Imports to a new database.": {
+            "$ref": "./examples/ImportNewDatabase.json"
+          },
+          "Imports to a new database, using private link for the SQL server and storage account.": {
+            "$ref": "./examples/ImportNewDatabaseWithNetworkIsolation.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ImportExportDatabaseDefinition": {
+      "description": "Contains the information necessary to perform import/export operation.",
+      "required": [
+        "storageKeyType",
+        "storageKey",
+        "storageUri",
+        "administratorLogin",
+        "administratorLoginPassword"
+      ],
+      "type": "object",
+      "properties": {
+        "databaseName": {
+          "description": "Name of the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "edition": {
+          "description": "Edition of the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "serviceObjectiveName": {
+          "description": "Service level objective name of the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "maxSizeBytes": {
+          "description": "Max size in bytes for the import database.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "storageKeyType": {
+          "description": "Storage key type.",
+          "enum": [
+            "SharedAccessKey",
+            "StorageAccessKey"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "StorageKeyType",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "storageKey": {
+          "description": "Storage key.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "storageUri": {
+          "description": "Storage Uri.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "administratorLogin": {
+          "description": "Administrator login name.",
+          "type": "string",
+          "x-ms-mutability": [
+            "create"
+          ]
+        },
+        "administratorLoginPassword": {
+          "description": "Administrator login password.",
+          "type": "string",
```

Comment:
>         "type": "string", [](start = 1, length = 26)

Isn't passing the server/DB password as string a sequrity risk? Many components in the control path may put these in their logs (may be in error paths).